osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmapSet.cs

```diff
@@ -210,12 +211,40 @@ public class FilterableDifficultyIcon : DifficultyIcon
         {
             private readonly BindableBool filtered = new BindableBool();
 
+            private OsuGame game;
+            private SongSelect songSelect;
+            private readonly BeatmapInfo info;
+
             public FilterableDifficultyIcon(CarouselBeatmap item)
                 : base(item.Beatmap)
             {
                 filtered.BindTo(item.Filtered);
                 filtered.ValueChanged += isFiltered => Schedule(() => this.FadeTo(isFiltered.NewValue ? 0.1f : 1, 100));
                 filtered.TriggerChange();
+
+                info = item.Beatmap;
+            }
+
+            protected override bool OnClick(ClickEvent e)
+            {
+                if (!filtered.Value)
+                {
+                    game.PresentBeatmap(info);
+
+                    if (e.AltPressed)
```

Comment:
this seems like weird / unnecessary behaviour.

Reply:
Should the `Alt` functionality be removed?

Reply:
Yeah, it's new behaviour which is weird and should be added as its own proposal, separate from this.